modules/sor/sorV2/lib/pools/composableStable/stablePoolV3.ts

```diff
@@ -0,0 +1,59 @@
+import { SwapKind, Token, TokenAmount } from '@balancer/sdk';
+import { ComposableStablePool } from './composableStablePool';
+import { Stable, StableState, Vault } from '@balancer-labs/balancer-maths';
+import { WAD } from '../../utils/math';
+import { BasePoolV3 } from '../basePool';
+
+export class StablePoolV3 extends ComposableStablePool implements BasePoolV3 {
```

Comment:
Do you think its useful to extend from `ComposableStablePool`? Feels like its mixing things up a bit and doesn't really offer any benefit.

Reply:
I use it to inherit the constructor/fromPrismaPool methods, so I get a BasePool, which is what the SOR expects as input. The alternative would be to copy/paste them here, but I think extending makes a bit more sense.
What do you think?

Reply:
It would probably be better to separate out the shared methods and use them where needed. Otherwise StablePoolV3 ends up inheriting a bunch of other stuff that isn't needed and may end up causing confusion elsewhere.

Reply:
I agree it may be confusing to inherit things that are not being used. I'll reimplement as separate pools. And I'll do the same with Weighted Pool V3, so it's easier to follow when we're maintaining the code.